testbed/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM golang:1.14.4-buster AS builder
+
+# PLAN_DIR is the location containing the plan source inside the container.
+ENV PLAN_DIR /plan
+
+# SDK_DIR is the location containing the (optional) sdk source inside the container.
+ENV SDK_DIR /sdk
+
+# Install brotli
```

Comment:
Installing brotli before any other code (such as `COPY . /`) that would invalidate the docker cache. We don't want to install this multiple times.